cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/BrokerFailureDetector.java

```diff
@@ -71,6 +71,8 @@ public BrokerFailureDetector(Queue<Anomaly> anomalies,
     _failedBrokersZkPath = config.getString(AnomalyDetectorConfig.FAILED_BROKERS_ZK_PATH_CONFIG);
     _anomalies = anomalies;
     _kafkaCruiseControl = kafkaCruiseControl;
+    _fixableFailedBrokerCountThreshold = config.getInt(AnomalyDetectorConfig.FIXABLE_FAILED_BROKER_COUNT_THRESHOLD_CONFIG);
```

Comment:
Isn't it sufficient to make this and `_fixableFailedBrokerCountThreshold ` a `Short`?